### DISCUSSION TITLE
missing UIBarButtonItem Xcode for list

### DISCUSSION THREAD

**Discussion by sendhelpp** [sst-inc/help#19] [Q&A - iOS] [answered]
[contact list v3.zip](https://github.com/sst-inc/help/files/6923967/contact.list.v3.zip)
Zipped up project because I can't get it all in a screenshot. please help thanks

**Comment by granwyntan** [comment, marked as answer]
Try tapping on the Main.storyboard canvas and thereafter tap the plus button again
<img width="793" alt="Screenshot 2021-08-03 at 21 12 06" src="https://user-images.githubusercontent.com/61010627/128021281-071ea642-7c7d-496d-bcab-98a3cde7b9e1.png">

Else, try quitting and reopening Xcode.
If you are still unable to find it, reinstall Xcode

**Reply by sendhelpp** [reply]
ok the bar appears in the mainstay board but doesn't when I launch the app, what configurations do I do to make it work

**Reply by sendhelpp** [reply]
mainstory*
